src/examples/dropdowns/MultiselectOverflow.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useEffect, useRef } from 'react';
+import { Dropdown, Multiselect, Field, Menu, Item, Label } from '@zendeskgarden/react-dropdowns';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import debounce from 'lodash.debounce';
+import { Tag } from '@zendeskgarden/react-tags';
+
+const options = [
```

Comment:
23 is quite a few options for the example. Would we feel ok bringing it down to like 10, 15?